scaleway/resource_instance_security_group_test.go

```diff
@@ -354,7 +350,6 @@ func TestAccScalewayInstanceSecurityGroup_RemovePort(t *testing.T) {
 			{
 				Config: `
 					resource "scaleway_instance_security_group" "base" {
-						tags = [ "test-terraform" ]
```

Comment:
Why are you changing this test? This is precisely what we want to test. Maybe you can add a test with:
- no tags
- adding tags
- modifying tags
- deleting tags

Reply:
Yes, I've just added a dedicated test only for tags on `TestAccScalewayInstanceSecurityGroup_Tags` and also for the other resource_instance_XXX with tags

Reply:
I've just added new test for this